types/ckeditor__ckeditor5-list/index.d.ts

```diff
@@ -1,16 +1,15 @@
-// Type definitions for @ckeditor/ckeditor5-list 29.0
+// Type definitions for @ckeditor/ckeditor5-list 32.0
 // Project: https://ckeditor.com/docs/ckeditor5/latest/api/list.html
 // Definitions by: Federico Panico <https://github.com/fedemp>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // Minimum TypeScript Version: 4.3
 
 export { default as List } from './src/list';
 export { default as ListEditing } from './src/listediting';
-export { default as ListProperties } from './src/listproperties';
-export { default as ListStyle } from './src/liststyle';
-export { default as ListStyleEditing } from './src/liststyleediting';
-export { default as ListStyleUI } from './src/liststyleui';
 export { default as ListUI } from './src/listui';
+export { default as ListProperties } from './src/listproperties';
+export { default as ListPropertiesEditing } from './src/listpropertiesediting';
+export { default as ListPropertiesUI } from './src/listpropertiesui';
```

Comment:
These files don't exist in the repository.
https://github.com/ckeditor/ckeditor5/tree/master/packages/ckeditor5-list/src

Reply:
They do in v32.

https://github.com/ckeditor/ckeditor5/tree/v32.0.0/packages/ckeditor5-list/src/